heat/core/dndarray.py

```diff
@@ -74,6 +79,74 @@ def device(self):
     def dtype(self):
         return self.__dtype
 
+    @property
+    def halo(self):
+        """
+        Returns
+        -------
+        int : number of rows or columns used as halo
```

Comment:
How would this work with arbitrary dimensions? In my opinion it would be good to pass an array with length equal to the number of dimensions everywhere that indicate the halo sizes in each dimensional direction

Reply:
As i understood it, it is only possible to split an tensor along one axis right? If so then there is only the need for one halo along that axis?

Reply:
You are absolutely right